doc/docs.md

```diff
@@ -2417,6 +2417,25 @@ NB: Each flag must go on its own line (for now)
 #flag linux -DIMGUI_IMPL_API=
 ```
 
+### #pkgconfig
+
+Add `#pkgconfig` directive is used to tell the compiler which modules should be used for compiling
+and linking using the pkg-config files provided by the respective dependencies.
+
+As long as backticks can't be used in `#flag` and spawning processes is not desirable for security
+and portability reasons, V uses its own pkgconfig library that is compatible with the standard
+freedesktop one.
+
+By default, if no flags are passed it will add `--cflags` and `--libs`, both lines below do the same:
```

Comment:
If you drop `By default`, this will be shorter than the line limit, and the CI will be green.

Reply:
Agreed. `By default, ` is not required because the rest of the sentence describes the default action.  Of course, it can also be handled by wrapping the line, but shorter sentences are often better anyway.